platform/darwin/src/MGLSource.h

```diff
@@ -0,0 +1,10 @@
+#import <Foundation/Foundation.h>
+
+@interface MGLSource : NSObject
+
```

Comment:
Should we document this interface (and its implementations, such as MGLGeoJSONSource)?